### ISSUE TITLE
Improve `ockam enroll` argument `--identity` (`clap` command)

### ISSUE THREAD

**Issue by nazmulidris** [build-trust/ockam#6117]
# Current behavior

The command `ockam enroll --identity <IDENTITY_NAME>` runs the command with the given identity name. And this is what you see when you run `ockam enroll --help`.

![Image](https://github.com/build-trust/ockam/assets/2966499/e40d9c00-02d6-49d6-96f9-1f14d854081e)

# Desired behavior

The `ockam enroll --help` command should provide some more information on the `--identity` flag. 

Add a comment to the field describing what it is: `/// Run the command as the given identity name`.

To make this change take a look at this [file](https://github.com/build-trust/ockam/blob/develop/implementations/rust/ockam/ockam_command/src/enroll/command.rs#L39). Add a doc comment w/ the text above to the `EnrollCommand::identity` field.

---

<i>We love helping new contributors! ❤️
If you have questions or need help as you explore, please join us on [Discord](https://discord.gg/bewvnm6zqS). If you're looking for other issues to contribute to, please checkout our [good first issues](https://github.com/build-trust/ockam/labels/good%20first%20issue).</i>

**Comment by axd99** [comment]
Hello nazmulidris, 

This is my first commit to github in a long time. I hope all is well with what and how  I have pushed back.

Cheers.

**Comment by nazmulidris** [comment]
@axd99 Hi 👋🏽 Would you like to work on this issue?

**Comment by nazmulidris** [comment]
@axd99 That's awesome, this is all yours. Please let us know if you have any questions as you explore. You can also ask questions on the contributors discord https://discord.gg/RAbjRr3kds

**Comment by axd99** [comment]
Hello @nazmulidris,

Thank you for assigning this to me.

Yes, I'd like to work on this.  

I created a pull request this morning (#6400)  and I reckon  it provides the fix.

Please let me know if I got that wrong, or anything that needs reworking.

Thank you.

**Comment by axd99** [comment]
On another note, I've tried to access the contributors discord on the link above, but I get an error message 'Unable to accept invite'. I have raised a ticket with the discord help desk, and they have come back to me (very quickly I must add) confirming that the link is valid but that my IP address must be banned (somehow) and that I should ask the moderator 'to clear all bans in Server Settings > Bans'.
I should then restart my client and try again.
Or it could also be (this is still from the help desk)  with my discord tag being in a black list (in a moderator bot?). I have joined Discord recently and have has very limited activity.
Any chance someone could have a look on your side?
